session/dto.go

```diff
@@ -22,12 +22,18 @@ import "github.com/mysteriumnetwork/node/identity"
 // ID represents session id type
 type ID string
 
+// BalanceTracker is responsible for interacting with the consumer in regards to payments
+type BalanceTracker interface {
+	Start() error
+	Stop()
+}
+
 // Session structure holds all required information about current session between service consumer and provider
 type Session struct {
-	ID         ID
-	Config     ServiceConfiguration
-	ConsumerID identity.Identity
-	Done       chan struct{}
+	ID              ID
+	ConsumerID      identity.Identity
+	Done            chan struct{}
+	DestroyCallback DestroyCallback
```

Comment:
Are we having a `DestroyCallback` back in Session struct?

Reply:
rebased a few times, so it must have snook back in somehow